test/integration/auth/node_test.go

```diff
@@ -245,6 +273,7 @@ func TestNodeAuthorizer(t *testing.T) {
 	// mirror pod and self node lifecycle is allowed
 	expectAllowed(t, createNode2MirrorPod(node2Client))
 	expectAllowed(t, deleteNode2MirrorPod(node2Client))
+	expectNotFound(t, createNode2MirrorPodEviction(node2Client))
```

Comment:
this seems odd... this section is testing that a node is allowed to create/delete its own mirror pods. I would have expected something like this:

```
// mirror pod and self node lifecycle is allowed
expectAllowed(t, createNode2MirrorPod(node2Client))
expectAllowed(t, deleteNode2MirrorPod(node2Client))
expectAllowed(t, createNode2MirrorPod(node2Client))
expectAllowed(t, createNode2MirrorPodEviction(node2Client))
...
```